### ISSUE TITLE
"Welcome!" should go

### ISSUE THREAD

**Issue by jsoref** [keepassxreboot/keepassxc#204]
## Expected Behavior
Don't show "Welcome!" as text.
Show something else...

## Current Behavior
"Welcome!" in the center of the view

## Possible Solution
Dunno. One approach is to show a list of recent files, or a wide button for creating a database, opening a database.
Also a button for "Import from {something}" -- something could be KeyChain.app, Firefox.app / Windows Credentials, Firefox.exe ...

## Steps to Reproduce (for bugs)
1. Open KeePassXC.app
2. Look at the center of main view

## Your Environment
* KeePassXC 2.1
* macOS

**Comment by TheZ3ro** [comment]
KeePassXC will open the recent file.
The welcome screen appears only when you haven't any recent file.

Maybe a screen with the following buttion:
- Open a DB file
- Create a new DB
- Import from KDB1

**Comment by droidmonkey** [comment]
I like the idea of showing an easy pick list of the recent files. Opening the recent files is an option "Load previous databases at startup". Adding the features you described @TheZ3ro is also very nice.

**Comment by jsha** [comment]
In #284, I suggested going straight to "Create new DB."

A quick review of other products:

 - 1Password presents three buttons: "I am new to ...", "I have used ... before", "Restore from backup", plus a link "Click here for the 10-minute Quick Start guide."
 - LastPass Chrome Extension prompts for email and master password, with a "Log In" button, and a "Create an Account" link below it.
 - Dashlane presents a splash screen saying "Never forget anotther password; Dashlane is the safest placeto store all your passwords and log in instantly, anywhere," plus a button [Create your free account] and a grey link [I already have an account].

I think the commonality we can extract here is: Most products give you a choice for new users and a choice for existing users. In two products, the choice for new users is emphasized by using a button, versus a link for the existing-user choice.

So this supports @TheZ3ro's idea of having a selection of three options on the welcome screen. I'd additionally suggest using color or other styling to emphasize the "New database" option.

![1Password splash](https://cloud.githubusercontent.com/assets/220205/22844011/f5ab5d7a-ef90-11e6-8676-19c01914741b.png)

![Dashlane splash](https://cloud.githubusercontent.com/assets/220205/22843994/dbf7b568-ef90-11e6-9ac8-d30e9ba83e18.png)

![LastPass splash](https://cloud.githubusercontent.com/assets/220205/22844031/0c17bcac-ef91-11e6-8ae8-f338dc244054.png)

**Comment by TheZ3ro** [comment]
I agree. I will start looking into this in the next few days

**Comment by TheZ3ro** [comment]
Something like this? @jsha 

![istantanea_2017-02-11_16-37-23](https://cloud.githubusercontent.com/assets/686894/22854982/75a5a5a2-f078-11e6-9266-1f87f0f0cd4e.png)

**Comment by jsha** [comment]
Looks awesome, thanks for working on this! A couple of thoughts / suggestions:

- It probably makes sense to keep "Recent databases" under a menu, since it's a power user feature (I expect most users to have just one database).
- It should probably say "Import new password database," "Create new password database," etc.
- How often do you expect "Import from KeePass1 database" to be used? I think if import from Firefox/Chrome/other password managers were available, it would be a good third button, but I think probably importing from KeePass1 is somewhat rare and it might be better to keep it to two buttons, since UI research shows that having more items on a page decreases success rates.

Also note that "database" shouldn't be capitalized.

**Comment by phoerious** [comment]
I think "Create new password database" is too much text.

**Comment by jsha** [comment]
Yeah, good point. The problem I'm trying to solve is that (I think) most users don't know about the "database" concept in KeePassXC, and probably think of a database as something run by engineers on large computers. Maybe "I am new to KeePassXC" and "I am an existing KeePassXC user?"

**Comment by phoerious** [comment]
I don't like that either. If you're new, you probably want to create a new DB, but only because you don't have one loaded right now, that doesn't mean you're new. This screen shouldn't only be useful for completely new users, but also for users who know their way around, but just don't have a database loaded right now.

We could rephrase it to "Create new password vault". That's a little shorter than "Create new password database", but introduces completely new terminology.

**Comment by TheZ3ro** [comment]
@jsha I'm working on displaying Recent databases only if there are some. The first time you open KeePassXC it will not be visibile since there won't be any recent database.

I was thinking about "database" being too generic but KeePass explicitly call it database
> "The program stores your passwords in a highly encrypted database. This database consists of only one file, so it can be easily transferred from one computer to another."

The "Import KeePass1" feature is common for people that were using KeePassX 0.4.x or KeePass1.x
The option is available also in the "Database" menu in the toolbar.
Another concept was this: https://imgur.com/a/9eGxv but the CSV option is disebled until we merge PR #146 

I've fixed the "database" capitalization in the last commit for #292 PR

**Comment by jsha** [comment]
> We could rephrase it to "Create new password vault". That's a little shorter than "Create new password database", but introduces completely new terminology.

I do think it would be nice to change up the terminology overall, but I agree that shouldn't be done haphazardly. Probably we should open a new issue for "How should we refer to databases?" Given that, I think it's probably fine to go with the present wording in the interests of landing the (very nice) redesign.

> I'm working on displaying Recent databases only if there are some. The first time you open KeePassXC it will not be visibile since there won't be any recent database.

This sounds great. And in the most common scenario, where the user has a single database, and always closes KeePassXC as opposed to closing the database, KeePassXC will automatically prompt for the password for that database on startup, so they won't even see this screen.

**Comment by TheZ3ro** [comment]
> This sounds great. And in the most common scenario, where the user has a single database, and always closes KeePassXC as opposed to closing the database, KeePassXC will automatically prompt for the password for that database on startup, so they won't even see this screen.
This is an already working option that can be enabled in the settings screen ("Remember last database"). In the case this is selected the last database will be automatically loaded without displaying the Welcome screen
